detect_secrets/plugins/keyword.py

```diff
@@ -26,22 +26,54 @@
 """
 from __future__ import absolute_import
 
+import re
+
 from .base import BasePlugin
 from detect_secrets.core.potential_secret import PotentialSecret
 from detect_secrets.plugins.core.constants import WHITELIST_REGEX
 
 
+# Note: All values here should be lowercase
 BLACKLIST = (
-    # NOTE all values here should be lowercase,
-    # otherwise _secret_generator can fail to match them
-    'pass =',
+    'apikey',
+    'api_key',
+    'pass',
     'password',
     'passwd',
-    'pwd',
+    'private_key',
     'secret',
     'secrete',
     'token',
 )
+# Uses lazy quantifiers
+BLACKLIST_REGEX = re.compile(
+    # Followed by double-quotes and a semi-colon
+    # e.g. private_key "something";
+    r'|'.join(
+        '(.*?){}(.*?)"(.*?)";'.format(
+            value,
+        )
+        for value in BLACKLIST
+    ) + '|' +
+    # Followed by a :
+    # e.g. token:
+    r'|'.join(
+        '(.*?){}[:](.*?)'.format(
```

Comment:
Why do we have a capturing class with only one element? Could also just do `{}:`